src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,26 @@
+@mixin onMobile {
+  @media (min-width: 320px) {
+    @content;
+  }
+}
```

Comment:
If we go mobile-first pattern, we don't need onMobile case. Mobile layout goes by default.